### DISCUSSION TITLE
Original Thunderbolt Firmware Package / Installer from Intel

### DISCUSSION THREAD

**Discussion by thegass** [osy86/HaC-Mini#1110] [Q&A]
Hi,

Does anybody still have the latest TB Firmware Package Intel released?
The morons at Asus are unable to provide it.

**Comment by Medizinmann-MD** [comment]
Hello,

I found this:
https://www.touslesdrivers.com/index.php?v_page=23&v_code=68832&v_langue=en

I laso found a reference for the ASUS Site to this driver but the link is down:
https://webcache.googleusercontent.com/search?q=cache:https://www.asus.com/sa-en/support/faq/1052574/

Did you try to email support directly?

**Reply by thegass** [reply]
thanks. the first link looks good.
will check the file next weekend if it's the correct one.

yes, i exchanged many mails with asus support. 
they said they didn't get that assets from intel.

**Reply by Medizinmann-MD** [reply]
Did you test it?

Does it work?
